crates/gitbutler-tauri/src/menu.rs

```diff
@@ -97,74 +124,132 @@ pub fn build(_package_info: &PackageInfo) -> Menu {
 
-    menu
+    // Help submenu
+    let mut help_menu = Menu::new();
+    // Adding a few common items to the Help menu
+    help_menu = help_menu.add_item(CustomMenuItem::new("help/documentation", "Documentation"));
+    help_menu = help_menu.add_item(CustomMenuItem::new("help/github", "Source Code"));
+    help_menu = help_menu.add_item(CustomMenuItem::new("help/release-notes", "Release Notes"));
+    help_menu = help_menu.add_native_item(MenuItem::Separator);
+    help_menu = help_menu.add_item(CustomMenuItem::new(
+        "help/share-debug-info",
+        "Share Debug Info…",
+    ));
+    help_menu = help_menu.add_item(CustomMenuItem::new("help/report-issue", "Report an Issue…"));
+    help_menu = help_menu.add_native_item(MenuItem::Separator);
+    help_menu = help_menu.add_item(CustomMenuItem::new("help/discord", "Discord"));
+    help_menu = help_menu.add_item(CustomMenuItem::new("help/youtube", "YouTube"));
+    help_menu = help_menu.add_item(CustomMenuItem::new("help/x", "X"));
+    help_menu = help_menu.add_native_item(MenuItem::Separator);
+    // add version info
+    help_menu = help_menu.add_item(disabled_menu_item(
+        "help/version",
+        format!("Version {}", _package_info.version).as_str(),
+    ));
+    // add as a submenu
+    menu = menu.add_submenu(Submenu::new("Help", help_menu));
 
-    //    #[allow(unused_mut)]
-    //    let mut menu = Menu::os_default(&package_info.name)
-    //
-    //    #[cfg(any(debug_assertions, feature = "devtools"))]
-    //    {
-    //        // Try to find the View menu and attach the dev tools item
-    //        let view_menu = menu.items.iter_mut().find(|item| match item {
-    //            MenuEntry::CustomItem(_) => false,
-    //            MenuEntry::Submenu(submenu) => submenu.title == "View",
-    //            MenuEntry::NativeItem(_) => false,
-    //        });
-    //
-    //        let devtools = CustomMenuItem::new("view/devtools", "Developer Tools");
-    //        if let Some(MenuEntry::Submenu(view_menu)) = view_menu {
-    //            view_menu.inner.items.push(devtools.into());
-    //        } else {
-    //            menu = menu.add_submenu(Submenu::new(
-    //                "Developer",
-    //                Menu::with_items([devtools.into()]),
-    //            ));
-    //        }
-    //    }
-    //
-    //    menu
+    menu
 }
 
+// Helper function to create a disabled menu item
 fn disabled_menu_item(id: &str, title: &str) -> CustomMenuItem {
     let mut item = CustomMenuItem::new(id, title);
     item.enabled = false;
     item
 }
 
+// Handle menu events and trigger corresponding actions
 pub fn handle_event<R: Runtime>(event: &WindowMenuEvent<R>) {
+    // DEVELOPER TOOLS MENU ITEM
     #[cfg(any(debug_assertions, feature = "devtools"))]
     {
         if event.menu_item_id() == "view/devtools" {
             event.window().open_devtools();
         }
     }
+
+    // HELP MENU ITEMS
+    if event.menu_item_id() == "help/documentation" {
```

Comment:
These `if` statements can be a single match statement.

```rust
let result = match event.menu_item_id() {
    "help/documentation" => open::that("..."),
    "something/else" => open::that("..."),
    unknown => panic!("unknown event emitted"),
};

if let Err(err) = result {
    tracing::error!(error = ?err, "failed to open url for {}", event.menu_item_id());
}
```

Something like that.